feature_engine/selection/base_selection_functions.py

```diff
@@ -1,17 +1,15 @@
-from typing import List, Union
 from types import GeneratorType
+from typing import List, Union
 
 import numpy as np
 import pandas as pd
 from sklearn.model_selection import cross_validate
 
-from feature_engine.variable_handling import (
-    check_all_variables,
-    check_numerical_variables,
-    find_all_variables,
-    find_numerical_variables,
-    retain_variables_if_in_df,
-)
+from feature_engine.variable_handling import (check_all_variables,
```

Comment:
could we put these changes back to the original representation? We use black for code styling.

Reply:
actually, I refer to all steps mentioned in [code contribution](https://feature-engine.trainindata.com/en/latest/contribute/contribute_code.html), and when I used isort and black on this file, it changed format

Reply:
I suspect difference can derive from whether we run isort first, then black or black first then isort

Reply:
yes, it comes from there. I normally run first isort and then black.